lib/rayyan-formats-plugins/plugins/medline.rb

```diff
@@ -119,7 +120,8 @@ def get_pubmed_url(pmid)
 
         def get_doi(article)
           article_ids = get_article_ids(article)
-          doi = article_ids&.detect { |aid| aid[:idtype] == :doi }
+          doi = article_ids&.detect { |lid| lid[:idtype] == :doi }
```

Comment:
Minor syntactic issue:
```suggestion
          doi = article_ids&.detect { |aid| aid[:idtype] == :doi }
```